### ISSUE TITLE
Add chance of at least 1 covid present and at least one new infection

### ISSUE THREAD

**Issue by bagiulia** [microcovid/microcovid#201]
I think some text that explains in %  how likely it is that at least somebody in the meeting has COVID and that at least someone else will get infected could be useful (for meetings with friends, where one might care about total risk to everyone) and very quick to add since the data is already there. Something like
(Right now)
The first part of the calculation is Person Risk: Each other person has a 1.746-in-a-million chance of currently having COVID.
This is a roughly 4,000-in-a-million (0.4%) chance of getting COVID from this activity with these people.
(Proposed)
The first part of the calculation is Person Risk: Each other person has a 1.746-in-a-million chance of currently having COVID. You selected 25 people near you, there's a X% chance of at least one of them having COVID
This is a roughly 4,000-in-a-million (0.4%) chance of getting COVID from this activity with these people. You selected 25 people near you, there's a X% chance of at least one of them getting infected.

**Comment by catherio** [comment]
This is pretty cool! I would want to make sure we weren't adding incomprehensibly much text.

**Comment by beshaya** [comment]
@bagiulia 

It's basically the same as "person risk" for small values. If you 25 people who each have 4000uCoV, the person risk is just 25 * 4000 = 10,000uCoV and the chances of at least one person having it is  (1- (1 - 4000e-6) ^ 25) = 95,000-in-a-million. The error is smaller for lower risks.

Likewise, if each person is subject to XuCoV, then the chance of one new infection is very close to 25X. Again, you start to see 5% error at activities that cost 10,000 uCoV, but for a first approximation this is quite good.

The latter isn't always true, because you can have asymmetric mask wearing (e.g. you're wearing a nice mask but not sure about everyone else).

**Comment by bagiulia** [comment]
I agree that it isn't much different (and surely easily obtainable) from what is already shown, however the person risk is only shown in the sentence  "The first part of the calculation is Person Risk: Each other person has a X-in-a-million chance of currently having COVID." which isn't as immediate as seeing a % or seeing a sentence that says "there's at least 1/X chance that with Y people, at least one has covid"
 Most people do struggle even with this kind of math, and so I think it would be useful if it was made more explicit ( the same for the chance of at least one new infection, even with the added uncertainty and error of asymmetric mask wearing)

**Comment by beshaya** [comment]
I agree that most people struggle with this kind of math. We need to balance that with humans having limited ability to parse lots of information on a screen - we already display 8 different numbers as results and sub-results.

I could support changing the copy of "Person Risk" to orient towards this:
`The first part of the calculation is Person Risk: Each other person has a 3,498-in-a-million chance of currently having COVID.` could perhaps be augmented to read as "the chance of at least one person in this group having covid is..." which is valid up to about 10% at which point we need to switch to expected value.

Perhaps something for UX research.
